### ISSUE TITLE
An error occurred while Layout/RescueEnsureAlignment

### ISSUE THREAD

**Issue by veganstraightedge** [rubocop-hq/rubocop#6551]
In a Dependabot version bump PR from `0.60` to to `0.61`, I get three new errors that weren't there before.

```
3 errors occurred:
An error occurred while Layout/RescueEnsureAlignment cop was inspecting …
An error occurred while Layout/RescueEnsureAlignment cop was inspecting …
An error occurred while Layout/RescueEnsureAlignment cop was inspecting …
```

The relevant methods are in this gist (with a comment point to the offending line).

https://gist.github.com/veganstraightedge/21fca151508247d364293fa51ef1b638

***

The common pattern between all three is this:

```ruby
foo =
  begin
    puts 'begin'
  rescue SomeKindOfException => e 
    puts 'rescue'
  end
```

The offending line is the `rescue …` line in all three cases.

Halp?

**Comment by Drenmi** [comment]
Duplicate of #6550. Please follow that issue for resolution. 🙇

**Comment by veganstraightedge** [comment]
Thank you!

**Comment by joshuapinter** [comment]
FYI, this was fixed in [0.61.1](https://github.com/rubocop-hq/rubocop/commit/54db2371b97385eabdf5ce1f719d62dfceb72bb5).
